package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "ember-tooltips",
```

Comment:
could you omit these `package.json` and `package-lock.json` changes?

Reply:
Yup, for sure.

We added some changes outside the scope of this PR so that I could publish my updates to NPM since installing from Github is not stable.

I can close this PR and make a new one with just the changes we actually want to make without my additional changes.